### PR TITLE
Make simple fields static

### PR DESCRIPTION
**Pull request by cmfcmf**
As with @harkenn's PR, the new fields currently aren't displayed as filters. Also, the adventure creation form is broken. However, the adventure edit form works just fine.

### CODE REVIEW

File: src/AppBundle/Command/AppElasticsearchReindexCommand.php [hunk 1]
```diff
@@ -45,18 +66,22 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
-                    'keyword' => [
-                        'type' => 'keyword',
-                        'ignore_above' => 256,
-                    ]
-                ]
-            ],
-            'slug' => [
-                'enabled' => false
-            ]
+            'setting' => self::FIELD_STRING,
```
Comment (cmfcmf) [inline src/AppBundle/Command/AppElasticsearchReindexCommand.php]: This change was missing in your PR to extract the setting table @harkenn. Only noticed it when doing my extraction today.

File: src/AppBundle/Entity/Adventure.php [hunk 2]
```diff
@@ -14,6 +14,7 @@
  * @ORM\Table(name="adventure")
  * @ORM\Entity(repositoryClass="AppBundle\Repository\AdventureRepository")
  * @UniqueEntity("title")
+ * @Gedmo\Loggable
```
Comment (cmfcmf) [inline src/AppBundle/Entity/Adventure.php]: I also made sure that all changes to an adventure are logged in the `ext_log_entries` table.

File: src/AppBundle/Entity/Adventure.php [hunk 3]
```diff
@@ -45,12 +47,114 @@ class Adventure
...
      * @ORM\Column(name="title", type="string", length=255, unique=true)
      * @Assert\NotBlank()
+     * @Gedmo\Versioned()
      */
     private $title;
 
     /**
      * @var string
      *
+     * @ORM\Column(type="text")
```
Comment (harkenn) [inline src/AppBundle/Entity/Adventure.php]: not nullable?

File: src/AppBundle/Entity/Adventure.php [hunk 4]
```diff
@@ -45,12 +47,114 @@ class Adventure
...
+     * @ORM\Column(type="integer", nullable=true)
+     * @Assert\Range(min=1)
+     * @Gedmo\Versioned()
+     */
+    private $numPages;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", length=255)
```
Comment (harkenn) [inline src/AppBundle/Entity/Adventure.php]: nullable?

File: src/AppBundle/Entity/AdventureDocument.php [hunk 5]
```diff
@@ -110,35 +110,53 @@ class AdventureDocument
      */
     private $foundIn;
 
-    public function __construct(int $id, string $setting, string $title, string $slug, array $info, float $score = 0.0)
+    public function __construct(
+        int $id,
+        string $setting,
+        string $title,
+        string $description,
```
Comment (harkenn) [inline src/AppBundle/Entity/AdventureDocument.php]: if we make description nullable, needs to be reflected here too, I'd guess? Should setting be null?

File: src/AppBundle/Form/AdventureType.php [hunk 6]
```diff
@@ -20,15 +24,92 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add('title', TextType::class, [
                 'help' => 'The title of the adventure. You can add more information in just a second.',
-                'required' => true
+                'required' => true,
+            ])
+            ->add('description', TextareaType::class, [
+                'help' => 'Description of the module.',
+                'required' => false,
```
Comment (harkenn) [inline src/AppBundle/Form/AdventureType.php]: If its not required, should its entity ORM annotation include `nullable`?

Comment (cmfcmf) [inline src/AppBundle/Form/AdventureType.php]: You're right, there wasn't much of a reason to not have them nullable. Changed in 269ed4b, also added the missing foundIn field to the edit form in 8edcd98.

**Pull request merged**